test/useDisposable.test.tsx

```diff
@@ -106,7 +106,6 @@ test("reactions run and dispose properly", async () => {
     expect(reaction2DisposerCalls).toBe(0) // this one is not early disposed
 
     rerender(<Component store={store} a={1} />)
-    flushEffects()
```

Comment:
they finally fixed that on react-testing-library? nice!

Reply:
It's more of the fix in React itself :) https://reactjs.org/docs/test-utils.html#act